apps/src/StudioApp.js

```diff
@@ -373,6 +373,10 @@ StudioApp.prototype.init = function(config) {
 
   if (this.isUsingBlockly()) {
     this.handleUsingBlockly_(config);
+  } else {
+    // handleUsingBlockly_ already does an onResize. We still want that goodness
+    // if we're not blockly
+    this.onResize();
```

Comment:
Would it make sense to just remove the call from `handleUsingBlockly_` and call it right after the `if`?